model/ExpandedLicensing/Individuals/NoneLicense.md

```diff
@@ -3,19 +3,18 @@ SPDX-License-Identifier: Community-Spec-1.0
 # NoneLicense
 
 ## Summary
+An Individual Value for License no license is present.
```

Comment:
I think there's a missing bit: e.g. "for License **where the SPDX data creator determines that** no license is present"

Reply:
Updated - along the same lines as the suggestion